.babelrc

```diff
@@ -1,17 +1,17 @@
 {
-  "presets": ["es2015", "react", "stage-1", "flow"],
+  "presets": ["env", "react"],
```

Comment:
"stage-1" is concerning since there's no guarantee that those things will be added into standard. Fortunately none of those specific uses seem to be failing.